modules/oxide/src/less/theme/components/toolbar-button/toolbar-split-button.less

```diff
@@ -10,6 +10,7 @@
   .tox-split-button__main {
     border-radius: @toolbar-button-border-radius 0 0 @toolbar-button-border-radius;
     margin-right: 0;
+    width: auto;
```

Comment:
Should there be a min-width and max-width? `tox-tbtn__select-label` should also have a max-width and handle text overflow? (check the convention)

Reply:
I disagree that truncating overflow text on buttons is a bad UX. It is a standard UI convention that prevents layout breakage. If requires, tooltip can be used as a fallback.